hooknode/services/iota.go

```diff
@@ -0,0 +1,53 @@
+package services
+
+import "github.com/iotaledger/giota"
+
+const seed = giota.Trytes("OYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRLOYSTERPRL")
+const security = 1
+const value = int64(0)
+const depth = int64(1)
+const mwm = int64(9)
+
+type IotaChunk struct {
+	Address string `json:"address"`
+	Value   int    `json:"value"`
+	Message string `json:"message"`
+	Tag     string `json:"tag"`
+}
+
+func AttachAndBroadcast(provider string, chunks *[]IotaChunk) error {
+	// TODO: Add logging to segment so we know what's going on.
+
+	api := giota.NewAPI(provider, nil)
+	oysterTag, _ := giota.ToTrytes("OYSTERGOLANG")
+	_, powFn := giota.GetBestPoW() // Log powName.
```

Comment:
In a future PR we could move oysterTag and powFn to some init() function.  They'll be the same everytime.

Reply:
I don't think we'd we won't gain much from it. Lambdas are ephemeral, so the go away once this function is called, and it won't stay in memory.

Reply:
Ok